src/messages/MClientReply.h

```diff
@@ -53,19 +53,23 @@ struct LeaseStat {
   __u32 duration_ms = 0;
   __u32 seq = 0;
   std::string alternate_name;
+  __u8 long_snap_name = 0;
```

Comment:
can we rename this to `is_long_snap_name` to indicate it's just a flag to determine whether the snapshot name needs to decrypted by the client?
 
can the data type be bool if it's a flag?

Reply:
> can we rename this to `has_long_snap_name` to indicate it's just a flag to determine whether the snapshot name needs to decrypted by the client?

Yeah, sounds good.

> 
> can __u8 and bool be interchangeably used?

Why I use `__u8` here is because It will be more clear how many bytes should the kclient need to decode from the buffer, the length of `bool` seems variable in cases ?

Reply:
Thanks, Xiubo. Got it!